Lesson13/cihan/script.js

```diff
@@ -0,0 +1,190 @@
+
+// 12. Round the number 3.7 down using Math.floor and 3.2 up using Math.ceil, log both.
+// CODE HERE
+console.log('Ex12', Math.floor(3.7));
+console.log('Ex12', Math.ceil(3.2)); 
+
+// 13. Declare a boolean variable isStudent = true. Log it.
+// CODE HERE
+let isStudent = true;
+console.log('Ex.13', isStudent);
+
+// 14. Initialize counter = 0, then increment it by 1 using counter++ and log it.
+// CODE HERE
+let counter = 0;
+counter++;
+console.log('Ex.14', counter);
+
+// 15. Initialize points = 10, add 5 to it using points += 5, then log points.
+// CODE HERE
+let points = 10;
+points += 5;
+console.log('Ex.15', points);
+
+// 16. Declare name="Alice", age=30, city="Paris". Log "Alice (30) lives in Paris" using template literals.
+// CODE HERE
+let name = 'Alice';
+let age = 30;
+let city = 'Paris';
+console.log(`${name} (${age}) lives in ${city}`); 
+
+// 17. Declare variables x=5, y=10, z=15. Log their total sum.
+// CODE HERE
+let x = 5;
+let y = 10;
+let z = 15;
+console.log('Ex.17', x + y + z);
+
+// 18. Declare dividend=10 and divisor=3. Log the quotient (divisionResult) and difference (differenceResult).
+// CODE HERE
+let dividend = 10;
+let divisor = 3;
+let divisionResult = dividend / divisor;
+let differenceResult = dividend - divisor;
+console.log('Ex.18', 'Quotient: ', divisionResult);
+console.log('Ex.18', 'Difference: ', differenceResult);
+
+// 19. Declare firstName and lastName. Create fullName by concatenating them with a space and log it.
+// CODE HERE
+let firstName = 'Cihan';
+let lastName = 'Kılıç';
+let fullName = firstName + ' ' + lastName;
+console.log('Ex.19', fullName);
+
+// 20. Declare firstFactor=7 and secondFactor=2. Log the product.
+// CODE HERE
+let firstFactor = 7;
+let secondFactor = 2;
+let product = firstFactor * secondFactor;
+console.log('Ex.20', product);
+
+// 21. Log the value of Math.PI.
+// CODE HERE
+console.log('Ex.21', Math.PI);
+
+// 22. Declare counter=0. Increment it using three different methods (e.g., counter++, counter+=1, counter=counter+1) and log the result each time.
+// CODE HERE
+let counter = 0;
+counter++;
+console.log('Ex.22', counter);
+counter += 1;
+console.log('Ex.22', counter);
+counter = counter + 1;
+console.log('Ex.22', counter);
+
+// 23. Declare initialTemperature=20. Increase it by 5 and log the result.
+// CODE HERE
+let initialTemperature = 20;
+initialTemperature += 5;
+console.log('Ex.23', initialTemperature);
+
+// 24. Declare numberEx9=6. Increment it using the prefix ++ operator and log both the variable and the incremented value.
+// CODE HERE
+let numberEx9 = 6;
+let incrementedValue = ++numberEx9;
+numberEx9 = ++numberEx9;
+console.log('Ex.24', numberEx9);
+console.log('Ex.24', incrementedValue);
+
+// 25. Declare numberEx10=8. Increment it using the postfix ++ operator and log both the original variable and the incremented value.
+// CODE HERE
+let numberEx10 = 8;
+let incrementedValuePostfix = numberEx10++;
+numberEx10 = numberEx10++;

```

Comment:
When using postfix increment, all you need to do is:
```suggestion
numberEx10++;
```

You don't need to reassign the value  (`numberEx10 =`) since `++` operator will change the value of the variable.

Replace
```JS
let incrementedValuePostfix = numberEx10++;
numberEx10 = numberEx10++;
```

with just
```JS
numberEx10++;
```